transform/macros/fact_event_type_monthly.sql

```diff
@@ -0,0 +1,34 @@
+{% macro fact_event_type_monthly(fact_table) -%} 
```

Comment:
Style: fix casing of SQL statements like SELECT, FROM, and GROUP BY to match uppercase convention used elsewhere.